Directory.Build.targets

```diff
@@ -35,9 +35,8 @@
     <!-- Prevent analyzer crashes from stopping things -->
```

Comment:
Lines 19-20 should have been deleted as well
![image](https://github.com/dotnet/extensions/assets/4403806/b4f6c000-6ba7-425a-9609-e59ad4265247)